Cargo.lock

```diff
@@ -1045,19 +1045,6 @@ dependencies = [
  "termcolor",
 ]
 
-[[package]]
-name = "env_logger"
-version = "0.8.1"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "54532e3223c5af90a6a757c90b5c5521564b07e5e7a958681bcd2afad421cdcd"
-dependencies = [
- "atty",
- "humantime 2.0.1",
- "log",
- "regex",
- "termcolor",
-]
-
```

Comment:
I believe this is introduced accidentally.

Reply:
`cargo update -p libc` did it.

Reply:
Ah, I see why, needed to update my submodules before cargo updating.

Reply:
Hm, `libc` don't depend on it so it should be unrelated, actually e852a4abf01f605aa71cd529887c17c4c64f33ed was reverted partially. I guess you didn't run `x.py check` or something to sync/update submodules.